arm-sql/2014-04-01/swagger/capabilities.json

```diff
@@ -0,0 +1,319 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "Azure Sql Database API spec",
+        "description": "The Azure SQL Database management API provides a RESTful set of web services that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases.",
+        "version": "2014-04-01"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "paths": {
+        "/subscriptions/{subscriptionId}/providers/Microsoft.Sql/locations/{locationName}/capabilities": {
+            "get": {
+                "tags": [
+                    "Capabilities"
+                ],
+                "operationId": "Capabilities_Get",
+                "description": "Returns information about the Azure SQL capabilities available for the specified region.",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "name": "locationName",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "description": "The name of the region for which the Azure SQL capabilities are retrieved."
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "",
+                        "schema": {
+                            "$ref": "#/definitions/LocationCapability"
+                        }
+                    }
+                }
+            }
+        }
+    },
+    "definitions": {
+        "MaxSizeCapability": {
+            "properties": {
+                "limit": {
+                    "readOnly": true,
+                    "type": "integer",
+                    "format": "int32",
```

Comment:
Internally we try to use the highest unit possible while still using whole numbers (2Gb rather than 2048Mb). However, to be safe, the user should be able to handle 2048Mb = 2Gb.